src/Kunstmaan/MediaBundle/Helper/File/SVGExtensionGuesser.php

```diff
@@ -3,9 +3,12 @@
 namespace Kunstmaan\MediaBundle\Helper\File;
 
 use Symfony\Component\HttpFoundation\File\MimeType\ExtensionGuesserInterface;
+use Symfony\Component\Mime\MimeTypes;
+
+@trigger_error(sprintf('The "%s" class is deprecated since KunstmaanMediaBundle 5.6, use "%s" instead.', SVGExtensionGuesser::class, MimeTypes::class), E_USER_DEPRECATED);
```

Comment:
```suggestion
@trigger_error(sprintf('The "%s" class is deprecated since KunstmaanMediaBundle 5.6 and will be removed in KunstmaanMediaBundle 6.0, use the "symfony/mime" component instead.', __CLASS__), E_USER_DEPRECATED);
```

Same for the message bellow